tools/Dashboard/DevHome.Dashboard/Views/DashboardView.xaml.cs

```diff
@@ -358,6 +382,28 @@ private async void RestorePinnedWidgets(object sender, RoutedEventArgs e)
                     Log.Logger()?.ReportError("DashboardView", $"RestorePinnedWidgets(): ", ex);
                 }
             }
+
+            // Now that we've ordered the widgets, put them in their final collection.
+            var finalPlace = 0;
+            foreach (var orderedWidget in restoredWidgetsWithPosition)

```

Comment:
nit: Since those two blocks are identical:
- Promote to a function
- Or,  `var restoredWidgets = restoredWidgetsWithPosition.Values.Concat(restoredWidgetsWithoutPosition.Values)`